src/Synapse/Synapse/help/New-AzSynapseSqlPool.md

```diff
@@ -55,6 +57,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AutoPauseTimer
```

Comment:
Commonly we would call it `-AutoPauseTimeout`

Reply:
Good point. I will change it soon in the next commit.

Reply:
I would suggest to keep the -AutoPauseTimer given the purpose of this parameter is set the value of the timer which could kick off the pausing operation. Changing it to AutoPauseTimeout lead to the feeling like this is the timeout of the auto-pause state which is not the purpose of this parameter. btw, this is Qixiao from the PM side for this part .

Reply:
Makes sense. Let me merge this PR